docs/api-reference/next.config.js/custom-page-extensions.md

```diff
@@ -4,46 +4,34 @@ description: Extend the default page extensions used by Next.js when resolving p
 
 # Custom Page Extensions
 
-Aimed at modules like [@next/mdx](https://github.com/vercel/next.js/tree/canary/packages/next-mdx), which adds support for pages ending with `.mdx`. You can configure the extensions looked for in the `pages` directory when resolving pages.
-
-Open `next.config.js` and add the `pageExtensions` config:
+You can extend the default Page extensions (`.tsx`, `.ts`, `.jsx`, `.js`) used by Next.js. Inside `next.config.js`, add the `pageExtensions` config:
```

Comment:
```suggestion
> **Note:** The upcoming [layouts and routing changes](https://nextjs.org/blog/layouts-rfc) will supersede this configuration option and will automatically be possible with the `app/` folder.

You can extend the default Page extensions (`.tsx`, `.ts`, `.jsx`, `.js`) used by Next.js. Inside `next.config.js`, add the `pageExtensions` config:
```

Reply:
Thoughts on this?

Reply:
> Thoughts on this?

Correct me if I was wrong, but the new Layout RFC mentions that only `.tsx`, `.ts`, `.jsx` and `.js` are supported by default. So I still have to configure `pageExtention` if I want to use `page.mdx` or `layout.mdx`.

The new layout and routing only supersedes `pageExtension` when putting non-page files (components, styles written in css-in-js, etc.) and page files together.